itou/approvals/models.py

```diff
@@ -942,54 +943,46 @@ class Prolongation(models.Model):
     # The addition of 0.25 day per year makes it possible to better manage leap years.
     MAX_DURATION = datetime.timedelta(days=365.25 * 10)
 
-    class Reason(models.TextChoices):
-        SENIOR_CDI = "SENIOR_CDI", "CDI conclu avec une personne de plus de 57 ans"
-        COMPLETE_TRAINING = "COMPLETE_TRAINING", "Fin d'une formation"
-        RQTH = "RQTH", "RQTH"
-        SENIOR = "SENIOR", "50 ans et plus"
-        PARTICULAR_DIFFICULTIES = (
-            "PARTICULAR_DIFFICULTIES",
-            "Difficultés particulières qui font obstacle à l'insertion durable dans l’emploi",
-        )
-        HEALTH_CONTEXT = "HEALTH_CONTEXT", "Contexte sanitaire"
-
     MAX_CUMULATIVE_DURATION = {
-        Reason.SENIOR_CDI.value: {
+        enums.ProlongationReason.SENIOR_CDI.value: {
```

Comment:
GH ne veut pas me laisser mettre un commentaire sur le fichier donc je prend la première ligne mais `ProlongationReason` étant un `TextChoice(str, enum.Enum)` il me semble qu'il n'y a pas besoin des `.value` pour `==`/`in`/etc, j'ai un doute pour le dict `MAX_CUMULATIVE_DURATION` mais ce n'est peut-être pas nécessaire non plus.

Même remarque pour _itou/approvals/tests/tests.py_.